com.flashforge.FlashPrint.json

```diff
@@ -0,0 +1,78 @@
+{
+  "app-id": "com.flashforge.FlashPrint",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "FlashPrint",
+  "tags": [
+    "proprietary"
+  ],
+  "finish-args": [
+    "--filesystem=home",
+    "--socket=x11"
```

Comment:
For X11, you need the `--share=ipc` as well.
And if this application uses OpenGL acceleration, you should also put the `--device=dri` here.